rust/noosphere/src/ffi/tracing.rs

```diff
@@ -0,0 +1,65 @@
+use noosphere_core::tracing::{initialize_tracing, NoosphereLog};
+use safer_ffi::prelude::*;
+
+const NOOSPHERE_LOG_SILENT: u32 = 0;
+const NOOSPHERE_LOG_BASIC: u32 = 1;
+const NOOSPHERE_LOG_CHATTY: u32 = 2;
+const NOOSPHERE_LOG_INFORMED: u32 = 3;
+const NOOSPHERE_LOG_ACADEMIC: u32 = 4;
+const NOOSPHERE_LOG_TIRESOME: u32 = 5;
+const NOOSPHERE_LOG_DEAFENING: u32 = 6;
+
+#[derive_ReprC(rename = "ns_noosphere_log")]
+#[repr(u32)]
+#[ffi_export]
+/// Configuration presets for Noosphere log behavior
```

Comment:
As this gets rendered via doxygen, and we use ints rather than enum for interface, consider mentioning this is used in `ns_tracing_initialize()` (can write as `"..log behavior, used in ns_tracing_initialize."`, no formatting or parens needed)